### DISCUSSION TITLE
translate in french

### DISCUSSION THREAD

**Discussion by weber158** [sumatrapdfreader/sumatrapdf#3079] [Documentation]
. annotations
. marque page

**Comment by GitHubRulesOK** [comment]
Je ne suis pas français, mais
![image](https://user-images.githubusercontent.com/12754913/200081895-8d72573b-962c-4bd3-b666-a26b978424e4.png)
![image](https://user-images.githubusercontent.com/12754913/200081957-a6f794d0-87ed-4263-9dd6-95bec0c60dcd.png)

**Comment by kjk** [comment]
Not sure what you mean but all translations are contributed by users as I don't know French or other languages.

If you want to improve french translations, you can do it via: https://www.apptranslator.org/app/SumatraPDF/fr

Currently it shows that all French strings are translated.

**Comment by Inisolia** [comment]
I'm a French! 

I've just download the tools! Will Try it and give me you  my feedback after. 

I can help for translate too btw .. I keep on' 

( Im looking for a thread talking about confidentiality CHART ) OR CLU
